### DISCUSSION TITLE
The unit files have no [Install] section. They are not meant to be enabled using systemctl.

### DISCUSSION THREAD

**Discussion by luispuerto** [qdm12/gluetun#1401] [Q&A]
Hey !

I'm trying to install make tun run on boot on a synology with DSM 7.1.1-42962 Update 4 and I get this message when I with the make it run `sudo systemctl enable /etc/systemd/system/tun.service` as [explained in the wiki](https://github.com/qdm12/gluetun/wiki/Synology-prerequisites).

This is the full message on terminal: 

```
sudo systemctl enable /etc/systemd/system/tun.service
Password:
The unit files have no [Install] section. They are not meant to be enabled
using systemctl.
Possible reasons for having this kind of units are:
1) A unit may be statically enabled by being symlinked from another unit's
   .wants/ or .requires/ directory.
2) A unit's purpose may be to act as a helper for some other unit which has
   a requirement dependency on it.
3) A unit may be started when needed via activation (socket, path, timer,
   D-Bus, udev, scripted systemctl call, …).
```

and the /etc/systemd/system/tun.service is: 

```
[Unit]
Description\=Run tun at startup
After\=network.target

[Service]
Type\=simple
User\=root
ExecStart\=/sbin/insmod /lib/modules/tun.ko
Restart\=on-failure

[Install]
WantedBy\=multi-user.target
```

Should I remove the install section?

**Comment by qdm12** [comment]
The Wiki for this was written a few years ago, maybe DSM changed since then?? 🤔 I cannot help since I have no DSM lying around, but let me know how you fix it so I can update the wiki 👍 You can probably just Google around on how to write a service on boot for dsm I guess?

**Reply by luispuerto** [reply]
Hey! 

Thanks for the swift reply! 

The funny thing is. I just updated DSM to the latest version, I had update 1 and updated to update 4, and before that message wasn't there… so perhaps it's something from the last version. 

I have no idea. I have some experience with Linux, but I'm new to DSM (I got the Synology like 2 weeks ago or so). 

PS/ Thanks a lot for providing Gluetun. It's awesome!

**Reply by luispuerto** [reply]
PS/ If I find the correct way to do it, it'll try to update the wiki if you want.
